src/components/Footer/Footer.js

```diff
@@ -1,5 +1,129 @@
+import FooterTop from './FooterTop';
+import styled from 'styled-components';
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <FooterDiv>
+      <Container>
+        <GlobalUl>
+          {FOOTER_DATA.map(data => {
+            return (
+              <FooterTop
+                id={data.id}
+                key={data.id}
+                data={data.data}
+                subdata={data.subdata}
+              />
+            );
+          })}
+          <li>
+            <FooterPtag>TALING CENTER</FooterPtag>
+            <FooterBboing>😊 뿌잉에게 문의하기</FooterBboing>
+            <li>
+              <FooterMarginTopPtag>운영시간</FooterMarginTopPtag>
+              <FooterMarginTopPtag>
+                평일 10:00 ~ 17:00 점심 13:00 ~14:00
+              </FooterMarginTopPtag>
+            </li>
+            <SnsImg alt="sns" src="images\free-icon-network-7185719.png" />
+            <SnsImg alt="sns" src="images\free-icon-youtube-1384012.png" />
+            <SnsImg alt="sns" src="images\premium-icon-facebook-665209.png" />
+            <SnsImg alt="sns" src="images\premium-icon-instagram-665211.png" />
+          </li>
+        </GlobalUl>
+        <FooterBottom>
+          <FooterInfo>
+            상호 : 뿌잉 | 주소 : 서울시 강남구 테헤란로 427 We Work tower 427
+            Ganamgu, Seoul, Korea
+          </FooterInfo>
+          <FooterInfo>
+            대표자명 : 김효정, 임재혁, 구본희, 이강호, 류미류, 김정수
+          </FooterInfo>
+        </FooterBottom>
+      </Container>
+    </FooterDiv>
+  );
 };
 
 export default Footer;
+
+const FooterDiv = styled.div`
+  margin-top: 30px;
+  padding: 40px 0 135px;
+  border-top: 1px solid #ccc;
+`;
+
+const Container = styled.div`
+  width: 1040px;
+  margin: 0 auto;
+`;
+
+const GlobalUl = styled.ul`
+  display: flex;
+`;
+
+const FooterBottom = styled.div`
+  color: ${props => props.theme.black};
```

Comment:
```suggestion
  color: ${({theme}) => theme.black};
```
구조분해할당 통해서 위와 같이 표현할수도 있습니다